src/views/open-api-docs-view/components/operation-header/index.tsx

```diff
@@ -15,9 +17,10 @@ import s from './operation-header.module.css'
 
 interface OperationHeaderProps {
 	slug: OperationProps['slug']
-	headerText: OperationProps['operationId']
+	headingSlot: ReactNode
+	headingAriaLabel: string
```

Comment:
Had to split out a separate `ariaLabel` here, for the heading permalink, as we can't render the now-with-word-breaks content into an `ariaLabel` prop 👍